docs/google/best_practices.md

```diff
@@ -201,3 +260,10 @@ print(list(flat_sweep.param_tuples()))
 #  (('<2**t - 1>', 0.681792830507429),),
 #  (('<2**t - 1>', 1.0),)]
 ```
+
+### Measure less qubits
```

Comment:
fewer

Reply:
also this might be subtly bad advice? I always have a sneaking suspicion that measuring spectator qubits could be useful. For example: what if you throw out any repetitions where spectator qubits are excited? I don't know if anyone's actually tried this though

Reply:
Yeah, I was on the fence for including this advice anyway.  I'll drop it.